hack/verify-golangci-lint.sh

```diff
@@ -129,13 +129,14 @@ kube::util::ensure-temp-dir
 # Installing from source (https://golangci-lint.run/welcome/install/#install-from-sources)
 # is not recommended, but for Kubernetes we prefer it because it avoids the need for
 # pre-built binaries for different platforms and gives more insights on dependencies.
-echo "installing golangci-lint and logcheck plugin from hack/tools into ${GOBIN}"
+echo "installing golangci-lint, logcheck and kubeapilinter plugins from hack/tools into ${GOBIN}"
```

Comment:
```suggestion
echo "installing golangci-lint, logcheck and kubeapilinter plugins from hack/tools/golangci-lint into ${GOBIN}"
```

Should have been changed already earlier, but better late than never...